Mesh_3/doc/Mesh_3/CGAL/Labeled_mesh_domain_3.h

```diff
@@ -233,6 +233,11 @@ The parameters are optional unless otherwise specified.
 <li> <b>`parameters::image` (mandatory)</b> the input 3D image. Must
 be a `CGAL::Image_3` object.
 
+<li> <b>`parameters::weights`</b> an input 3D image that provides
+weights associated to each voxel (the word type is `unsigned char`,
+and the voxels values are integers between 0 and 255).
+The weights image can be generated with `CGAL::Mesh_3::generate_weights()`.
```

Comment:
I think you should add that the dimensions of the `weights` image are the same as the dimensions of the `image`. I know that is a triviality, but in a reference manual it should be added.